src/test/groovy/org/shipkit/internal/gradle/util/StringUtilTest.groovy

```diff
@@ -0,0 +1,14 @@
+package org.shipkit.internal.gradle.util
+
+import spock.lang.Specification
+
+class StringUtilTest extends Specification {
+
+    def "knows if empty"() {
+        expect:
+        StringUtil.isEmpty(null)
+        StringUtil.isEmpty("")
+        !StringUtil.isEmpty(" ")
+        !StringUtil.isEmpty("xx")
```

Comment:
Very true :)

It is a very simple logic, 4 lines, I'm not convinced spock tables would add value.